lib/cpp/README.md

```diff
@@ -55,15 +54,22 @@ you are using libthriftnb you will also need libevent.
 
 ## Dependencies
 
-boost shared pointers
-http://www.boost.org/libs/smart_ptr/smart_ptr.htm
+If your C++ environment implements C++11 or later, boost will automatically use
```

Comment:
Even if boost itself provides mapping to `std::` pointers, next sentence suggests that boost is not needed. Some little clean up could help confused users ;) (this is also first place in README.md where boost is mentioned, hence the potential confusion)